# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1387655891 18000
#      Sat Dec 21 14:58:11 2013 -0500
# Branch stable
# Node ID f1fa148766d623f7f24cf541ea8b88d8c1b63d33
# Parent  20f2b3c48c4c658fb774e320867d368400d87174
3.8.0 release candidate 2

* configure.ac (OCTAVE_VERSION): Bump to 3.8.0-rc2.
(OCTAVE_RELEASE_DATE): Set to 2013-12-21.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -14,23 +14,23 @@ dnl Copyright (C) 1993-2013 John W. Eato
 ### FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 ### for more details.
 ### 
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <http://www.gnu.org/licenses/>.
 
 AC_PREREQ([2.62])
-AC_INIT([GNU Octave], [3.8.0-rc1], [http://octave.org/bugs.html], [octave])
+AC_INIT([GNU Octave], [3.8.0-rc2], [http://octave.org/bugs.html], [octave])
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 OCTAVE_API_VERSION_NUMBER="49"
 OCTAVE_API_VERSION="api-v$OCTAVE_API_VERSION_NUMBER+"
-OCTAVE_RELEASE_DATE="2013-12-04"
+OCTAVE_RELEASE_DATE="2013-12-21"
 OCTAVE_COPYRIGHT="Copyright (C) 2013 John W. Eaton and others."
 AC_SUBST(OCTAVE_VERSION)
 AC_SUBST(OCTAVE_API_VERSION_NUMBER)
 AC_SUBST(OCTAVE_API_VERSION)
 AC_SUBST(OCTAVE_RELEASE_DATE)
 AC_SUBST(OCTAVE_COPYRIGHT)
 
 dnl FIXME: We should auto-insert the Mercurial changeset ID into the
